erpnext/config/hr.py

```diff
@@ -198,6 +198,19 @@ def get_data():
 				},
 			]
 		},
+		{
+			"label": _("Employee Template"),
```

Comment:
Instead of adding to a different section, maybe we can add these templates to Employee Lifecycle itself? Employee Template doesn't seem like a proper label as well 🤔